### DISCUSSION TITLE
Notification sound

### DISCUSSION THREAD

**Discussion by larrasket** [Syncplay/syncplay#696] [Ideas]
It would be helpful if there could be a notification sound, which can be configurable from the settings (enable/disable).  This is extra helpful when you are not watching but you would like to observe comments every once and a while.

**Comment by Et0h** [comment]
Thanks for your suggestion.

Syncplay is not intended to be a general purpose chat application, and so your requested feature is not likely to be one that we will implement, but it is useful to know how people use our software.

As noted at #315 there are a variety of reasons why we do not try to do everything as we only have a small developer team and are reluctant to increase the complexity of the software.

I anticipate that developing, testing and maintaining audio support across a lot of operating systems (including many flavours of Linux) would be quite a commitment.
